mlflow/R/mlflow/R/model-registry.R

```diff
@@ -45,3 +121,194 @@ mlflow_delete_registered_model <- function(name, client = NULL) {
     data = list(name = forge::cast_string(name))
   )
 }
+
+#' List registered models
+#'
+#' Retrieves a list of registered models.
+#'
+#' @param max_results Maximum number of registered models to retrieve.
+#' @param page_token Pagination token to go to the next page based on a previous query.
+#' @template roxlate-client
+#' @export
+mlflow_list_registered_models <- function(max_results = 100, page_token = NULL, client = NULL) {
+  client <- resolve_client(client)
+
+  response <- mlflow_rest(
+    "registered-models",
+    "list",
+    client = client,
+    verb = "GET",
+    version = "2.0",
+    query = list(
+      max_results = max_results,
+      page_token = page_token
+    )
+  )
+
+  return(list(
+    registered_models = response$registered_model,
+    next_page_token = response$next_page_token
+  ))
+}
+
+#' Get latest model versions
+#'
+#' Retrieves a list of the latest model versions for a given model.
+#'
+#' @param name Name of the model.
+#' @param stages A list of desired stages. If the input list is NULL, return
+#'               latest versions for ALL_STAGES.
```

Comment:
I think it's standard to indent two spaces only.

Reply:
Done.